README.md

```diff
@@ -67,6 +67,7 @@ To style multiple tracks of the same type, you can use the `cssClass` property.
     git clone https://github.com/hammerlab/pileup.js.git
     cd pileup.js
     npm install
+    npm run prod
```

Comment:
Sure -- I actually added it there because I thought you liked the `prod` tasks :)